file_impl.rb:80: warning: Using the last argument as keyword parameters is deprecated

Index: lib/puppet/file_system/file_impl.rb
--- lib/puppet/file_system/file_impl.rb.orig
+++ lib/puppet/file_system/file_impl.rb
@@ -77,7 +77,7 @@ class Puppet::FileSystem::FileImpl
   end
 
   def read(path, opts = {})
-    path.read(opts)
+    path.read(**opts)
   end
 
   def read_preserve_line_endings(path)
